packages/client/src/formValidation.js

```diff
@@ -0,0 +1,36 @@
+module.exports = {
+  emailValidation: function (email) {
+    if (
+      /^[a-zA-Z0-9.!#$%&’*+/=?^_`{|}~-]+@[a-zA-Z0-9-]+(?:\.[a-zA-Z0-9-]+)*$/.test(
+        email
+      )
```

Comment:
It'd probably be worth bringing in a library not to have to deal with this. email testing is tough if not impossible.

Reply:
Thank you!  We were thinking "try it DIY" but switching to Formik/Yup